src/main/java/com/khomishchak/ws/controllers/ExchangerController.java

```diff
@@ -44,22 +45,25 @@ public List<Balance> getAccountBalances(@RequestAttribute long userId) {
         return exchangerService.getAllMainBalances(userId);
     }
 
-    @DeleteMapping("/{exchangerCode}/balance")
-    public DeleteExchangerResp deleteAccountBalance(@RequestAttribute long userId, @PathVariable String exchangerCode) {
-        return exchangerService.deleteExchangerForUser(userId, ExchangerCode.valueOf(exchangerCode));
+    @DeleteMapping("/{balanceId}/balance")
```

Comment:
Incorrect path name.
```suggestion
    @DeleteMapping("/balances/{balanceId}")
```